src/main/java/uk/gov/companieshouse/chdorderconsumer/service/CHDOrderService.java

```diff
@@ -1,13 +1,18 @@
 package uk.gov.companieshouse.chdorderconsumer.service;
 
+import org.springframework.http.HttpStatus;
```

Comment:
See http://code-analysis.platform.aws.chdev.org:9000/project/issues?id=uk.gov.companieshouse%3Achd-order-consumer&pullRequest=10&resolved=false&types=CODE_SMELL.

Reply:
Ty fixed all the smelly code